write-fonts/generated/generated_ift.rs

```diff
@@ -28,6 +28,7 @@ impl Ift {
         uri_template_length: u16,
         uri_template: Vec<u8>,
         patch_format: u8,
+        optional_charstring_offsets: Vec<u8>,
```

Comment:
yuck.

A few options:

- you could use something like `#[compile_type(OptionalCharstringOffsets)]` and then add that type to wrtie-fonts; it would just wrap two `Option<u32>` and write them if they were present.
- ... actually that's probably the best option. It feels hacky, though, and I'd prefer trying to fix this in one of the larger ways mentioned above.

but requiring this to be a buffer of pre-encoded bytes is definitely not great API.